ports/mdl/vcpkg.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "mdl",
+  "version-semver": "2021.1.2",
```

```suggestion
  "version": "2021.1.2",
```

This version scheme does not confirm with semver because it has the year in its version.

---

Good catch, I have integrated your suggestion into my last commit.